GliaWidgets/Sources/ViewModel/Call/CallViewModel.swift

```diff
@@ -112,6 +112,7 @@ class CallViewModel: EngagementViewModel, ViewModel {
             showConnecting()
             let operatorName = interactor.engagedOperator?.firstName ?? Localization.Engagement.defaultOperator
             action?(.setOperatorName(operatorName))
+            action?(.showSnackBarView(text: "App screen is visible to the agent"))
```

Comment:
Should this text be localized string instead?

Reply:
fixed - b87be880d55d623a783b6bc4f32c680b87c7564d